scripts/globals/mobskills.lua

```diff
@@ -116,6 +116,9 @@ end
 -- dmgMod   : linear multiplier for damage (1 default)
 -- tpEffect : Defined in xi.mobskills.physicalTpBonus
 -----------------------------------
+-- TODO: Reduce complexity
+-- Disable cyclomatic complexity check for this function:
+-- luacheck: ignore 561
```

Comment:
You've expanded two blocks that look exactly the same at a glance. Break them out into a helper function with multiple args and multiple returns and you'll be able to get rid of this exception

Reply:
> You've expanded two blocks that look exactly the same at a glance. Break them out into a helper function with multiple args and multiple returns and you'll be able to get rid of this exception

Thanks for the suggestion, I broke out the logic to a helper function as suggested and updated the PR.